src/record/sdk/playback.cpp

```diff
@@ -364,6 +373,15 @@ k4a_result_t k4a_playback_seek_timestamp(k4a_playback_t playback_handle,
     return K4A_RESULT_SUCCEEDED;
 }
 
+uint64_t k4a_playback_get_recording_length_usec(k4a_playback_t playback_handle)
+{
+    RETURN_VALUE_IF_HANDLE_INVALID(0, k4a_playback_t, playback_handle);
+
+    k4a_playback_context_t *context = k4a_playback_t_get_context(playback_handle);
+    RETURN_VALUE_IF_ARG(0, context == NULL);
+    return context->last_timestamp_ns / 1000;
```

Comment:
now that there is file time stamp and device timestamp, it is not clear to me what last_timestamp_ns refers to.

Reply:
All timestamps internal to the playback API (like this one), are still relative to start of file. This could probably be refactored to reflect this now that we have 2 time domains, but for now I think that can be left for another PR, since it's not visible to users of the API.

Reply:
can you just rename last_timestamp_ns to something like last_file_timestamp_ns?

Reply:
Done